### ISSUE TITLE
Pattern attribute value is not a valid regular expression

### ISSUE THREAD

**Issue by reggi** [dahlia/hollo#80]
```
Pattern attribute value ^[\\p{L}\\p{N}._\\-]+$ is not a valid regular expression: Uncaught SyntaxError: Invalid regular expression: /^[\\p{L}\\p{N}._\\-]+$/v: Invalid character in character class
```

![Image](https://github.com/user-attachments/assets/aba58b9b-d2a5-46fc-9918-f19085b6290e)

**Comment by reggi** [comment]
it took some time but did generate the account

**Comment by dahlia** [comment]
Fixed in 0.3.5.
